test/unit/matrix/test_panel.cpp

```diff
@@ -245,7 +245,7 @@ void testAccess(const GlobalElementSize size, const TileElementSize blocksize,
 
   // rw-access
   for (const auto& idx : panel.iteratorLocal()) {
-    start_detached(panel.readwrite(idx) | then([idx](auto&& tile) {
+    start_detached(panel.readwrite(idx) | then([idx](auto const& tile) {
```

Comment:
```suggestion
    start_detached(panel.readwrite(idx) | then([idx](const auto& tile) {
```

Reply:
This habit may die hard so you may have to remind me in the future as well :P Hopefully not too often.

https://github.com/eth-cscs/DLA-Future/pull/766/commits/737c518331717bde2645f5b3fffbc793d9bb3250

I also changed a couple of other `auto&&`s that I noticed in the same file.